### ISSUE TITLE
Drumsequencer with gridrows set to 1 and drum presets

### ISSUE THREAD

**Issue by therealfakemoot** [BespokeSynth/BespokeSynth#820]
## Describe the bug
If you take a `drumsequencer` and reduce its `gridrows` to 1, I expected selecting one of the presets from the list to only fill out one row. Actual behavior is that the full sequence comes out of the `drumsequencer`.
![image](https://user-images.githubusercontent.com/602852/163752980-e44bd8dd-1d33-401b-9740-a415e56531f8.png)

## Build info

bespoke NIGHTLY 1.1.0 (Apr 18 2022 02:55:32) - Windows 10 - HEAD 213d9ac

## Steps To Reproduce
See screenshot above. Select "amen" (or really any of the presets, i think they all use multiple rows )

## Expected vs. actual behavior
Basically, changing `gridrows` seems to be a purely visual change, the sequencer still has those sequencer rows available for modification. Ideally, reducing the gridrows wouldn't produce surprising notes from rows that ostensibly don't exist.
